msdk-featdet/msdk-featdet-ADAP-3D/src/test/java/io/github/msdk/featdet/ADAP3D/common/algorithms/BiGaussianTest.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * (C) Copyright 2015-2017 by MSDK Development Team
+ *
+ * This software is dual-licensed under either
+ *
+ * (a) the terms of the GNU Lesser General Public License version 2.1 as published by the Free
+ * Software Foundation
+ *
+ * or (per the licensee's choosing)
+ *
+ * (b) the terms of the Eclipse Public License v1.0 as published by the Eclipse Foundation.
+ */
+package io.github.msdk.featdet.ADAP3D.common.algorithms;
+
+import java.io.File;
+import java.io.FileWriter;
+
+import org.apache.commons.collections4.map.MultiKeyMap;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import io.github.msdk.MSDKException;
+import io.github.msdk.datamodel.rawdata.RawDataFile;
+import io.github.msdk.io.netcdf.NetCDFFileImportMethod;
+
+public class BiGaussianTest {
+	
+	  private static final String TEST_DATA_PATH = "src/test/resources/";
```

Comment:
You should use class resources instead of a hard-coded path like this. See [```MzMLFileParserTest.java```](https://github.com/msdk/msdk/blob/master/msdk-io/msdk-io-mzml2/src/test/java/io/github/msdk/io/mzml2/MzMLFileParserTest.java) for an example.